packages/components/forma-36-react-timepicker/package.json

```diff
@@ -39,9 +34,8 @@
     "@types/lodash.orderby": "^4.6.6",
     "@types/react": "^16.9.11",
     "@types/react-dom": "^16.9.3",
+    "eslint-plugin-prettier": "^3.1.4",
```

Comment:
Is this needed?

Reply:
When I was linting (or building) the time picker `tsdx` telling me that `eslint-plugin-prettier` could not be found, but this was before I learned about the `nohoist` config so maybe we can use that instead

What I can say is that apparently `tsdx` does not understand workspaces well right now

I will post the error we get here so we can all see. And I will try to find the open issue in `tsdx` related to the error
maybe you all can help me with alternatives 🙌

Reply:
So this is what we get when I remove this and run `tsdx lint`:
```
Error: Failed to load plugin 'prettier' declared in 'BaseConfig': Cannot find module 'eslint-plugin-prettier'
Require stack:
- /Users/gui.santos/Contentful/forma-36/packages/components/forma-36-react-timepicker/__placeholder__.js
Referenced from: BaseConfig
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.resolve (internal/modules/cjs/helpers.js:33:19)
    at Object.resolve (/Users/gui.santos/Contentful/forma-36/node_modules/eslint/lib/shared/relative-module-resolver.js:44:50)
    at ConfigArrayFactory._loadPlugin (/Users/gui.santos/Contentful/forma-36/node_modules/eslint/lib/cli-engine/config-array-factory.js:959:39)
    at ConfigArrayFactory._loadExtendedPluginConfig (/Users/gui.santos/Contentful/forma-36/node_modules/eslint/lib/cli-engine/config-array-factory.js:779:29)
    at ConfigArrayFactory._loadExtends (/Users/gui.santos/Contentful/forma-36/node_modules/eslint/lib/cliengine/config-array-factory.js:725:29)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/gui.santos/Contentful/forma36/node_modules/eslint/lib/cli-engine/config-array-factory.js:660:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/Users/gui.santos/Contentful/forma36/node_modules/eslint/lib/cli-engine/config-array-factory.js:596:20)
    at _normalizeObjectConfigData.next (<anonymous>)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Does anyone have any clue?

Reply:
tsdx should install `eslint-plugin-prettier` for us. Is it being hoisted or did you add it to `nohoist`?

Reply:
I'm getting it even when I do this:

```
"nohoist": [
      "@contentful/forma-36-fcss/node-sass-import",
      "**/eslint-plugin-prettier",
      "**/eslint-plugin-prettier/**"
    ]
```

Should I put tsdx in this list instead? It's only used in timepicker and date picker right now

Reply:
yep! I think not hoisting tsdx worked

I have to say, this is a con for tsdx. It should work with workspaces

Reply:
There's this, but doesn't seem like there's much actual information there: https://github.com/formium/tsdx/issues/275

Reply:
Yep! I saw this issue before. I think it's ok to not hoist tsdx for now. It's used only in date and timepicker

wdyt?

Reply:
I agree, but I wish I understood why it's giving us problems 🙂 [Formik is a monorepo](https://github.com/formium/formik) using tsdx as well and they don't have to jump through hoops to get it working. Maybe it'll get better once `react-components` also use tsdx.

Reply:
Let's keep this in mind and try to go back to an ideal setup in the next PR's 🙌

Reply:
@mshaaban0 this is why we need tsdx to not be hoisted